### ISSUE TITLE
Icons in buttons in footer not properly styled

### ISSUE THREAD

**Issue by Shurelia** [GeekyAnts/NativeBase#1576]
## react-native, react and native-base version
RN: 0.51.0,
React: 16.2.0
NativeBase: 2.3.7

## Expected behaviour
`variables.topTabBarActiveTextColor` changes the color of icons in my footer, including icons that are in buttons.
## Actual behaviour
`variables.topTabBarActiveTextColor` does not change the color of icons in buttons in the footer, or at least not icons in transparent buttons
## Steps to reproduce (code snippet or screenshot)
For a footer that looks like this:
```
<Footer>
  <Right>
    <Button transparent>
       <Icon name="map" ios="ios-globe-outline" android="md-globe" />
    </Button>
  </Right>
<Footer>
```
The icons will always be black, as no styling is attributed to them.
## Is the bug present in both ios and android or in any one of them?
Both
## Any other additional info which would help us debug the issue quicker.
PR: #1577

**Comment by SupriyaKalghatgi** [comment]
@Shurelia `topTabBarActiveTextColor` this refers to **Tabs**
Your problem description says 
> Icons in buttons in footer

You seem to be confused here

**Comment by Shurelia** [comment]
This is the color already styling the rest of footer. I didn't write this,
I'm just sticking with what's already there for consistency.

**Comment by Shurelia** [comment]
@SupriyaKalghatgi To be clear, `topTabBarActiveTextColor` is the color _already used by nativebase_ to style the code. It confused me probably as much as it confused you, but it's the precedent already set by nativebase. If you don't like the implementation of the fix, that's fine, please say as much, but the issue of icons inside buttons inside footers not being styled is very much a bug and this issue is not invalid.

**Comment by Shurelia** [comment]
@SupriyaKalghatgi For reference: https://github.com/GeekyAnts/NativeBase/blob/master/src/theme/components/Footer.js

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.3.9](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.9)
